source/process/training.rst

```diff
@@ -767,3 +767,37 @@ This is restricted to people with an `@mattermost.com` email address.  It is mea
 Partners team
 ---------------------------------------------------------
 This is an invite only team meant for Mattermost staff and partners.  This is a team for staff to interact with partners and for partners to interact with other partners. You should prefer public channels when you want partner-to-partner interactions and private channels when you want staff-to-partner interactions.  Remember that any partner added to the team will be able to join any public channel.
+
+---------------------------------------------------------
+Creating New Jira Bug Tickets
+---------------------------------------------------------
+
+A bug is any “obvious error”. If you’re not reporting an obvious error, please file a Story ticket instead. Errors on `unsupported platforms <https://docs.mattermost.com/install/requirements.html>`_ are not considered bugs.
+
+1. Confirm you’re filing a new issue
+---------------------------------------------------------
+
+ - Search existing tickets in Jira to confirm your issue isn’t already filed by someone else.
```

Comment:
Will people need more guidance about how/where to search in JIRA? Could say something such as `in the Mattermost project`, but then that does risk making the doc more brittle and at risk of becoming out of date, even without a specific URL (which was my first thought).